demo/long_video_demo.py

```diff
@@ -70,6 +112,7 @@ def show_results(model, data, label, args):
 
     msg = 'Preparing action recognition ...'
     text_info = {}
+    out_json = {}
     fourcc = cv2.VideoWriter_fourcc(*'mp4v')
     frame_size = (frame_width, frame_height)
 
```

Comment:
in line#120, `video_writer` should be `None` when `args.out_file` is not a Video